src/piggui.rs

```diff
@@ -78,10 +80,18 @@ impl Sandbox for Gpio {
             }
         };
 
+        let num_pins = GPIO_DESCRIPTION.len();
+
+        // TODO Fixing default PinFunction for each pin
```

Comment:
I see now - your using GROUND as a temporary selection when no selection is made.
Could you use 
- "" or
- "Not Configured" 
instead?

Reply:
Yaa i added it as TODO, will do that today

Reply:
OK, will you add it to this PR?

Reply:
Sure, will do